addons/addons.json

```diff
@@ -1,5 +1,6 @@
 [
   "editor-searchable-dropdowns",
+  "blocklydevtools",
```

Comment:
I meant between `reorder-custom-inputs` and `NEW ADDONS ABOVE THIS` but it's not a big deal as long as it's not all the way at the bottom.

Reply:
Ok, moved it.